src/models/user.js

```diff
@@ -0,0 +1,20 @@
+import Sequelize from 'sequelize';
+import { sequelize } from '../config/config'
+
+const User = sequelize.define('users', {
+name:{
+    type: Sequelize.STRING
+},
+email:{
+    type: Sequelize.STRING,
+    unique: true
+},
+phone:{
+   type: Sequelize.STRING,
+   unique: true
+}
+},{
+   timestamps:false
+});
+
```

Comment:
you should consider the model that allows association because we will need to use relations in tables

Reply:
on it

Reply:
You need to provide an explicit and detailed description of your PR. you can refer to this [link](https://github.com/andela/bestpractices/wiki/Git-naming-conventions-and-best-practices)